include/lsst/jointcal/PhotometryTransfo.h

```diff
@@ -18,19 +24,17 @@ class PhotometryTransfoSpatiallyInvariant;
 /*
  * A photometric transform, defined as a scale factor of the input calibration.
  *
- *     initialCalibFlux (Maggies) * transfo(x,y) -> correctedFlux (Maggies)
- *
- * @todo Eventually will be defined the same as PhotoCalib:
- *     instFlux (ADU) -> flux (maggies)
+ *     inputFlux (ADU or Maggies) * transfo(x,y) -> correctedFlux (Maggies)
```

Comment:
How can ADU * factor -> Maggies? Which is what this comment seems to imply.

Reply:
That's the same definition as is used in PhotoCalib.

If the part you don't like is that inputFlux could be ADU or maggies, a photometryTransfo doesn't actually care what the input units are, which is used in composed transforms (e.g. ADU->intermediate->maggies).

Reply:
Yes, that was my point.

Reply:
I've clarified the docs.